### PR TITLE
Unify headings titles and breadcrumbs for SDK 3.0.2 - closes#479

### LINKED ISSUE #479: Unify headings, titles and breadcrumbs for SDK 3.0.2
- Same page title as the navigation title of the page for every page
- Update navigations to not use bold navigation links
reference/configuration reference: change title to be only "configuration"
- add banner images for Elements, Commander (like in the README files on GitHub)
- Use imperative form in page titles, e.g.: Config file handling --> Handle config files
- On the start page of every version, change the H0 title to Introduction, remove the subtitle Introduction if present
- Use sentence case in titles (small letters except first letter of first word, names, abbreviations,...)
- Change titles to remove redundancy: 
  - `Reference/Configuration reference` to `Reference/Configuration`
  ... and so on

### PR DESCRIPTION
**Pull request by chris529**
closes #479

### CODE REVIEW

File: modules/ROOT/pages/reference/api.adoc [hunk 1]
```diff
@@ -8,14 +8,14 @@
 TIP: Check the xref:{url_core_api}[Lisk Core API] to make live API requests to the Lisk xref:{url_core_intro}[public networks].
 
 [[_overview]]
-== Access Restrictions
+== Access restrictions
```
Comment (Tschakki) [inline modules/ROOT/pages/reference/api.adoc:11]: As already mentioned in the SDK 2.3.7 PR, this content is auto generated, so will be overridden in future ;)

File: modules/ROOT/pages/reference/lisk-commander/index.adoc [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
 :toc:
-:imagesdir: ../assets/images
+:imagesdir: ../../../assets/images
 
 :url_commander_commands: reference/lisk-commander/user-guide/commands.adoc
 :url_commander_user_guide: reference/lisk-commander/user-guide.adoc
 :url_nodejs_download: https://nodejs.org/en/download/
 :url_nodejs: https://nodejs.org/
 
+image:diagram_sdk.png[Logo]
```
Comment (Tschakki) [inline modules/ROOT/pages/reference/lisk-commander/index.adoc]: Please use this banner: https://github.com/LiskHQ/lisk-sdk/tree/development/commander

File: modules/ROOT/pages/reference/lisk-elements/index.adoc [hunk 3]
```diff
@@ -1,35 +1,37 @@
 = Overview
 Mona Bärenfänger <mona@lightcurve.io>
-:description: The Lisk Elements overview provides an introduction to the Lisk Elements library and the different sub-packages. The overall basic usage and how to upgrade with NPM is described here..
+:description: The Lisk elements overview provides an introduction to the Lisk elements library and the different sub-packages. The overall basic usage and how to upgrade with NPM is described here.
```
Comment (Tschakki) [inline modules/ROOT/pages/reference/lisk-elements/index.adoc]:
```suggestion
:description: The Lisk Elements overview provides an introduction to the Lisk elements library and the different sub-packages. The overall basic usage and how to upgrade with NPM is described here.
```

Lisk Elements is the product name :) Please capitalize, see also the other occurrences on the page.

File: modules/ROOT/pages/reference/lisk-elements/packages/api-client.adoc [hunk 4]
```diff
@@ -1,6 +1,6 @@
 = API client
 Mona Bärenfänger <mona@lightcurve.io>
-:description: Technical references regarding the API Client package of Lisk Elements. This consists of usage examples, available options and example responses.
+:description: Technical references regarding the API client package of Lisk elements. This consists of usage examples, available options and example responses.
```
Comment (Tschakki) [inline modules/ROOT/pages/reference/lisk-elements/packages/api-client.adoc]:
```suggestion
:description: Technical references regarding the API client package of Lisk Elements. This consists of usage examples, available options and example responses.
```

File: modules/ROOT/pages/reference/lisk-framework/index.adoc [hunk 5]
```diff
@@ -1,8 +1,8 @@
 = Overview
 Mona Bärenfänger <mona@lightcurve.io>
-:description: The Lisk Framework overview provides a high-level overview of the Lisk Framework architecture, including it's modules and components, how they communicate and how to change the default configuration.
+:description: The Lisk framework overview provides a high-level synopsis of the Lisk framework architecture, including it's modules and components, how they communicate, and also how to change the default configuration.
```
Comment (Tschakki) [inline modules/ROOT/pages/reference/lisk-framework/index.adoc]:
```suggestion
:description: The Lisk Framework overview provides a high-level synopsis of the Lisk Framework architecture, including it's modules and components, how they communicate, and also how to change the default configuration.
```

Please fix everywhere

File: modules/ROOT/pages/tutorials/transport1.adoc [hunk 6]
```diff
@@ -364,7 +364,7 @@ nano light-alarm.js
 
 Now insert the code of the `LightAlarmTransaction`.
 Use kbd:[CMD+V] to paste the contents in the file.
-In order to save and exit `nano`, use the following keys:
+In order to save and exit `nano`, using the following keys:
```
Comment (Tschakki) [inline modules/ROOT/pages/tutorials/transport1.adoc]: I think it was correct like it was, no change needed

File: modules/ROOT/pages/reference/lisk-elements/index.adoc [hunk 7]
```diff
@@ -1,35 +1,37 @@
...
+:imagesdir: ../../../assets/images
 :page-previous: /lisk-sdk/reference/lisk-commander/index.html
 :page-previous-title: Lisk Commander
 
 :url_nodejs: https://nodejs.org/
 :url_nodejs_download: https://nodejs.org/en/download/
 :url_elements_packages: reference/lisk-elements/packages/index.adoc
 
-Lisk Elements is a JavaScript library containing several separately installable npm packages, each covering a distinct region of Lisk-related functionality.
+image:lisk_docu_blog_elements.png[Logo]
```
Comment (Tschakki) [inline modules/ROOT/pages/reference/lisk-elements/index.adoc]: Please use same banner image as in lisk-sdk repo

**Review by Tschakki — CHANGES_REQUESTED** [review]
Please see my comments, and also check the points in the issue description again, e.g.

- nav.adoc still has bold links .  OK,  this one is now fixed..
- title redundancy not removed (see config reference)  - done.
- unify headings and nav links: not all are unified (see e.g. lisk transport tutorial)  - also done..

File: modules/ROOT/pages/reference/lisk-commander/index.adoc (hunk 2 above)
Comment (chris529) [reply modules/ROOT/pages/reference/lisk-commander/index.adoc]: OK, now the correct logo is inserted :-)

File: modules/ROOT/pages/tutorials/transport1.adoc (hunk 6 above)
Comment (chris529) [reply modules/ROOT/pages/tutorials/transport1.adoc]: OK, let's leave it as it is.

File: modules/ROOT/pages/reference/lisk-elements/index.adoc (hunk 7 above)
Comment (chris529) [reply modules/ROOT/pages/reference/lisk-elements/index.adoc]: OK, it is done..

File: modules/ROOT/pages/reference/changelog.adoc [hunk 8]
```diff
@@ -88,7 +88,7 @@ process.exit(0);
 
 == Updated genesis block for devnet
 
-Due to the changes that have been introduced for the structure of transaction objects, the genesis block and it's transactions were generated newly for Lisk SDK version 3.0.
+Due to the changes that have been introduced for the structure of transaction objects, the genesis block and it's transactions were recently generated for Lisk SDK version 3.0.
```
Comment (Tschakki) [inline modules/ROOT/pages/reference/changelog.adoc]:
```suggestion
Due to the changes that have been introduced for the structure of transaction objects, the genesis block and it's transactions were created new for Lisk SDK version 3.0.
```

File: modules/ROOT/pages/reference/lisk-commander/user-guide/commands.adoc [hunk 9]
```diff
@@ -520,11 +520,11 @@ EXAMPLE
 }
 ----
 
-=== Encrypt Message
+=== Encrypt message
 
 This encrypts a message for a given recipient public key using your secret passphrase.
 
-This command uses lisk-elements passphrase module to encrypt a message you provide for a given public key using a randomly generated nonce.
+This command uses Lisk-elements passphrase module to encrypt a message you provide for a given public key using a randomly generated nonce.
```
Comment (Tschakki) [inline modules/ROOT/pages/reference/lisk-commander/user-guide/commands.adoc]:
```suggestion
This command uses Lisk Elements passphrase module to encrypt a message you provide for a given public key using a randomly generated nonce.
```

File: modules/ROOT/pages/reference/lisk-elements/index.adoc [hunk 10]
```diff
@@ -1,35 +1,37 @@
-= Overview
+= Lisk Elements
 Mona Bärenfänger <mona@lightcurve.io>
-:description: The Lisk Elements overview provides an introduction to the Lisk Elements library and the different sub-packages. The overall basic usage and how to upgrade with NPM is described here..
+:description: The Lisk Elements overview provides an introduction to the Lisk elements library and the different sub-packages. The overall basic usage and how to upgrade with NPM is described here.
```
Comment (Tschakki) [inline modules/ROOT/pages/reference/lisk-elements/index.adoc]: Lisk Elements is the product name, please capitalize everywhere.

```suggestion
:description: The Lisk Elements overview provides an introduction to the Lisk Elements library and the different sub-packages. The overall basic usage and how to upgrade with NPM is described here.
```

File: modules/ROOT/pages/tutorials/transport.adoc [hunk 11]
```diff
@@ -27,7 +27,7 @@ This is done to ensure that it has not been manipulated during the transport.
 
 The carrier will be rewarded upon the successful transportation, and penalized if the transportation has failed for some reason.
 
-For convenient usage of the system, a client application called Lisk Transport will be provided that visualizes all relevant information regarding the supply chain tracking system. In turn the Lisk Transport application enables the users to interact with the system by posting custom transactions.
+For convenient usage of the system, a client application called Lisk transport will be provided that visualizes all relevant information regarding the supply chain tracking system. In turn the Lisk transport application enables the users to interact with the system by posting custom transactions.
```
Comment (Tschakki) [inline modules/ROOT/pages/tutorials/transport.adoc]:
```suggestion
For convenient usage of the system, a client application called Lisk Transport will be provided that visualizes all relevant information regarding the supply chain tracking system. In turn the Lisk Transport application enables the users to interact with the system by posting custom transactions.
```

Lisk Transport is the name of the app, please update everywhere

File: modules/ROOT/pages/tutorials/transport1.adoc (hunk 6 above)
Comment (Tschakki) [reply modules/ROOT/pages/tutorials/transport1.adoc]: can you then please undo the change?  Now done..

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/reference/lisk-commander/index.adoc [hunk 12]
```diff
@@ -1,15 +1,17 @@
...
-:imagesdir: ../assets/images
+:imagesdir: ../../../assets/images
 
 :url_commander_commands: reference/lisk-commander/user-guide/commands.adoc
 :url_commander_user_guide: reference/lisk-commander/user-guide.adoc
 :url_nodejs_download: https://nodejs.org/en/download/
 :url_nodejs: https://nodejs.org/
 
-Lisk Commander is a Command Line Interface tool designed to be a low entry point and the most convenient tool for developers who are looking for an efficient way of coding.
+image:banner_commander.png[Logo]
```
Comment (Tschakki) [inline modules/ROOT/pages/reference/lisk-commander/index.adoc:12]: Please check image

File: modules/ROOT/pages/reference/lisk-elements/index.adoc [hunk 13]
```diff
@@ -1,35 +1,37 @@
...
+:imagesdir: ../../../assets/images
 :page-previous: /lisk-sdk/reference/lisk-commander/index.html
 :page-previous-title: Lisk Commander
 
 :url_nodejs: https://nodejs.org/
 :url_nodejs_download: https://nodejs.org/en/download/
 :url_elements_packages: reference/lisk-elements/packages/index.adoc
 
-Lisk Elements is a JavaScript library containing several separately installable npm packages, each covering a distinct region of Lisk-related functionality.
+image:banner_elements.png[Logo]
```
Comment (Tschakki) [inline modules/ROOT/pages/reference/lisk-elements/index.adoc:13]: Image missing

**Review by Tschakki — CHANGES_REQUESTED** [review]
Please check the banner images again - did you upload them to the images folder?

File: modules/ROOT/pages/reference/lisk-commander/index.adoc (hunk 12 above)
Comment (chris529) [reply modules/ROOT/pages/reference/lisk-commander/index.adoc:12]: OK, I think it has now worked :-)

File: modules/ROOT/pages/reference/lisk-elements/index.adoc (hunk 13 above)
Comment (chris529) [reply modules/ROOT/pages/reference/lisk-elements/index.adoc:13]: It should now be OK

**Review by Tschakki — APPROVED** [review]
Thank you!

**Pull request merged**